src/main/java/school/hei/haapi/service/StudentFileService.java

```diff
@@ -32,8 +33,8 @@ public byte[] generatePdf(String studentId, String template) {
 
   private Context loadContext(String studentId) {
     ClassPathResourceResolver pathResolver = new ClassPathResourceResolver();
```

Comment:
why is it a method field ?

Reply:
it's to access to apply method

Reply:
use it as an instance atrribute in the StudentFileService Class